expyfun/io/__init__.py

```diff
@@ -1,4 +1,4 @@
 # -*- coding: utf-8 -*-
 from ._wav import read_wav, write_wav
-from ._hdf5 import read_hdf5, write_hdf5
+from .._externals._h5io import read_hdf5, write_hdf5
```

Comment:
This won't work properly because of the `title` arg, which used to effectively be `expyfun`. You need to do something like:
```
from .._externals._h5io import read_hdf5 as _read_hdf5, write_hdf5 as _write_hdf5

def read_hdf5(fname):
    """..."""
    return _read_hdf5(fname, title='expyfun')

def write_hdf5(...)
```